src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -4160,7 +4160,24 @@ internal void TraceVariableSet(string varName, object value)
             // because 'ToStringParser' would iterate through the enumerator to get the individual elements, which will
             // make irreversible changes to the enumerator.
             bool isValueAnIEnumerator = PSObject.Base(value) is IEnumerator;
-            string valAsString = isValueAnIEnumerator ? typeof(IEnumerator).Name : PSObject.ToStringParser(_context, value);
+            string valAsString = string.Empty;
+            if (isValueAnIEnumerator)
+            {
+                valAsString = typeof(IEnumerator).Name;
```

Comment:
Isn't this just `nameof(IEnumerator)`?

Reply:
I think the intention is to get runtime name.